client/src/app/dashboard/datatable-notif/datatable-notif.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { MatPaginator, MatSort } from '@angular/material';
+import { DatatableNotifDataSource } from './datatable-notif-datasource';
+
+@Component({
+  selector: 'app-datatable-notif',
+  templateUrl: './datatable-notif.component.html',
+  styleUrls: ['./datatable-notif.component.css']
+})
+export class DatatableNotifComponent implements OnInit {
```

Comment:
this class somehow needs to know for which component it needs to receive the notifications. a type variable might be necessary here.